plugins/available/man.plugin.bash

```diff
@@ -11,4 +11,4 @@ about-plugin 'colorize man pages for better readability'
 
 : "${LESS:=}"
 export "${!LESS_TERMCAP@}"
-export LESS="R${LESS#-}"
+export LESS="--raw-control-chars ${LESS}"
```

Comment:
```suggestion
export LESS="--RAW-CONTROL-CHARS ${LESS}"
```

`--raw-control-chars` (`-r`) and `--RAW-CONTROL-CHARS` (`-R`) are different. The use of `--raw-control-chars` (`-r`) is not recommended by `less` (as you the warning in the man page of `less`). With `--raw-control-chars` (`-r`), it also shows the warning messages like "may be a binary file.  See it anyway?" if the man page produces escape sequences that `less` does not recognize, which would be annoying.